### DISCUSSION TITLE
Granular documentation on Grafana Internal Metrics

### DISCUSSION THREAD

**Discussion by vinodiyerrbx** [grafana/grafana#37131] [General]
The current documentation on the available Grafana Internal Metrics is limited to only the configuration changes needed to enable the metrics (https://grafana.com/docs/grafana/latest/administration/view-server/internal-metrics/).

This is a feature request to elaborate on the metrics which will be available once the configuration change is made to enable internal metrics. For example, ***grafana_datasource_request_duration_seconds*** internal metric represents the full round trip time from the Grafana Server, to the Datasource, and back to the Grafana Server, etc.

**Comment by soni-kanishk** [comment]
any updates on this?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
